db/local_db/stop_database.go

```diff
@@ -22,7 +22,7 @@ type StopStatus int
 
 const (
 	// ServiceStopped :: StopStatus - service was stopped
-	ServiceStopped StopStatus = iota
+	ServiceStopped StopStatus = iota + 10
```

Comment:
comment to explain why +10

also if you come across code with this style of comment please update to 
```
ServiceStopped indicates service was stopped
```

Reply:
fixed